test/test-utils.js

```diff
@@ -58,3 +56,5 @@ export function deleteIndexedDB(dbName) {
         throw e;
     });
 }
+
+export const sleep = (ms) => new Promise((resolve) => { setTimeout(resolve, ms); });
```

Comment:
I moderately wonder if it's better to use `function` instead for this kind of stuff (for legibility and ease of understanding, as constants have a certain expected use case).